src/diagnostics/land_compute_methods.jl

```diff
@@ -131,6 +131,20 @@ end
 @diagnostic_compute "soil_net_radiation" SoilCanopyModel p.soil.R_n
 @diagnostic_compute "soil_temperature" SoilCanopyModel p.soil.T
 
+function compute_soil_albedo!(
+    out,
+    Y,
+    p,
+    t,
+    land_model::SoilCanopyModel{FT},
+) where {FT}
+    if isnothing(out)
+        return FT(0.5) .* p.soil.PAR_albedo .+ FT(0.5) .* p.soil.NIR_albedo
+    else
+        @. out = FT(0.5) .* p.soil.PAR_albedo .+ FT(0.5) .* p.soil.NIR_albedo
```

Comment:
with the @. in front, can remove the broadcasts in the rest of the expression

Reply:
done